daffodil-test/src/test/scala/org/apache/daffodil/section10/representation_properties/encodings.tdml

```diff
@@ -0,0 +1,275 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
It looks like this file is a duplicate of the same file in src/test/resources, but with the addition of the sigle IBM1047 element and test case. The new element/parserTestCase should be moved to the encodings.tdml file in src/test/resources and this file deleted.

Reply:
I verified by comparison inside Emacs that this file not only duplicates `daffodil-test/src/test/resources/org/apache/daffodil/section10/representation_properties/encodings.tdml` but also has unwanted changes at line 54 (a stray backtick character was inserted) and lines 239 & 255 (a no-break space, \u00A0, was replaced with a regular space, \u0020).  @susmita-m145 , please delete this duplicate file and use an UTF8-compliant text editor to carefully insert your new lines into the original file without changing any other lines.

At line 55, insert:

```xml
  <xs:element name="eString1047" type="xs:string" dfdl:encoding="IBM-1047"/>
```

At line 119, insert:

```xml
  <tdml:parserTestCase name="IBM1047" root="eString1047" model="enc1" 
    description="IBM-1047 basic test">
    <tdml:document>
      <tdml:documentPart type="text" encoding="IBM-1047">12345</tdml:documentPart>
    </tdml:document>
    <tdml:infoset>
      <tdml:dfdlInfoset>
        <tns:eString>12345</tns:eString>
      </tdml:dfdlInfoset>
    </tdml:infoset>
  </tdml:parserTestCase>
```

Then please run `sbt test` and verify that the tests pass before you push the new commit.

Reply:
Wow, I'm not the only guy who uses emacs? How great is that?  :-) 

@susmita-m145  This team is *really good* and I depend on this team as well as our automated continuous-integration runs, to catch things. I have fallen into the trap of running all the tests, seeing success, then trying to slide in one more little inconsequential change, like just updating a code-comment, and then pushing it, only to find *blam* I made a typo and things are broken. 

As you are one of the people working on EBCDIC/IBM stuff, I am also very interested in your use case for Daffodil and the priority you give to finishing the tickets like the P, V text number pattern features (for COBOL data) etc. 

I am hoping you have a bit more time to push this PR over the finish line. Thanks -mikeb